scripts/docs.py

```diff
@@ -369,25 +370,34 @@ def update_config(lang: str):
         config = get_base_lang_config(lang)
         config["nav"] = current_config["nav"]
         config["theme"]["language"] = current_config["theme"]["language"]
-    languages = [{"en": "/"}]
-    alternate: List[Dict[str, str]] = config["extra"].get("alternate", [])
-    alternate_dict = {alt["link"]: alt["name"] for alt in alternate}
-    new_alternate: List[Dict[str, str]] = []
+
+    languages: List[Dict[str, str]] = []
+    alternate: List[Dict[str, str]] = []
+
+    # Language names sourced from https://quickref.me/iso-639-1. FastAPI
```

Comment:
This is a very minor complaint, but the dot in the end of the sentence breaks the URL.

Reply:
Ok, I've added a quick commit to fix it.